src/neo/Ledger/PoolItem.cs

```diff
@@ -26,11 +26,17 @@ internal class PoolItem : IComparable<PoolItem>
         /// </summary>
         public DateTime LastBroadcastTimestamp;
 
+        /// <summary>
+        /// Cache Tx's FeePerByte to avoid recompute when comparing
+        /// </summary>
+        public long TxFeePerByteCache;
```

Comment:
`FeePerByte` is a single division. I don't think there will be any improvement in caching it.